features/net/FEATURE_IPV4/TESTS/mbedmicro-net/tcp_client_hello_world/main.cpp

```diff
@@ -66,17 +67,19 @@ int main() {
         TEST_ASSERT_TRUE(found_200_ok);
         TEST_ASSERT_TRUE(found_hello);
 
-        if (!found_200_ok) result = false;
-        if (!found_hello) result = false;
+        if (found_200_ok && found_hello) result = true;
 
         printf("HTTP: Received %d chars from server\r\n", ret);
         printf("HTTP: Received 200 OK status ... %s\r\n", found_200_ok ? "[OK]" : "[FAIL]");
         printf("HTTP: Received '%s' status ... %s\r\n", HTTP_HELLO_STR, found_hello ? "[OK]" : "[FAIL]");
-        printf("HTTP: Received massage:\r\n\r\n");
+        printf("HTTP: Received message:\r\n");
         printf("%s", buffer);
+        sock.close();
+    }
+    else {
```

Comment:
Last little nit pick: could you put the above two lines on the same line? According to our committer guide, we follow the K&R style: https://github.com/ARMmbed/mbed-os/blob/master/docs/COMMITTERS.md#rules